dask-gateway/dask_gateway/client.py

```diff
@@ -407,22 +407,40 @@ def list_clusters(self, status=None, **kwargs):
         """
         return self.sync(self._clusters, status=status, **kwargs)
 
-    async def _cluster_options(self):
+    def _config_cluster_options(self):
+        opts = dask.config.get("gateway.cluster.options")
+        return {k: format_template(v) for k, v in opts.items()}
+
+    async def _cluster_options(self, disable_local_defaults=False):
         url = "%s/gateway/api/clusters/options" % self.address
         req = HTTPRequest(url=url, method="GET")
         resp = await self._fetch(req)
         data = json.loads(resp.body)
-        return Options._from_spec(data["cluster_options"])
+        options = Options._from_spec(data["cluster_options"])
+        if not disable_local_defaults:
+            options.update(self._config_cluster_options())
+        return options
 
-    def cluster_options(self, **kwargs):
+    def cluster_options(self, disable_local_defaults=False, **kwargs):
         """Get the available cluster configuration options.
 
+        Parameters
+        ----------
+        disable_local_defaults : bool, optional
```

Comment:
I'm not 100% happy with this kwarg name, if anyone has a better idea (cc @ericdill, @mrocklin). Other ideas:
- `use_local_defaults`
- `no_local_defaults`

Reply:
At one point Aaron Meurer recommended to me that all options should be positively named, so as to avoid confusing double-negatives.  That policy would encourage the name `allow_local_defaults` instead.

Reply:
Renamed to `use_local_defaults` instead.